config/webpack.config.api.js

```diff
@@ -73,4 +73,26 @@ module.exports = ({
     new webpack.DefinePlugin(env.individuallyStringified),
   ],
   mode: process.env.NODE_ENV,
-} /*: any */);
+};
+
+const client = {
+  ...baseConfig,
+  target: "web",
+  output: {
+    ...baseConfig.output,
+    path: `${paths.apiBuild}/client`,
+    libraryTarget: "umd",
+  },
+};
+
+const server = {
+  ...baseConfig,
+  target: "node",
+  output: {
+    ...baseConfig.output,
+    path: `${paths.apiBuild}/server`,
+    libraryTarget: "commonjs",
+  },
+};
+
+module.exports = [client, server];
```

Comment:
This needs an explicit type annotation or flow is not happy on my machine: 
```suggestion
module.exports = ([client, server] /*: Array<any>*/);
```

Reply:
Flow passes in CI, wondering why it's failing on your machine. Reluctant to cast to any if it's not absolutely necessary. Do you maybe have an old version of flow or something?

Reply:
Now that [types-first](https://flow.org/en/docs/lang/types-first/) is the default (as of version 0.134), file exports cannot be implicit `any`. 
```
❯ yarn flow --version
yarn run v1.22.10
$ flow --version
Flow, a static type checker for JavaScript, version 0.135.0
Done in 0.19s.
```